.github/workflows/pythonapp.yml

```diff
@@ -237,3 +237,45 @@ jobs:
           pytest -v
           test/test_cvedb.py
           test/test_cli.py
+
+  cve_scan:
+    name: CVE Scan on dependencies
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Get Date
+        id: get-date
+        run: |
+          echo "::set-output name=date::$(/bin/date -u "+%Y%m%d")"
+        shell: bash
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: get cached python packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: get cached database
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/cve-bin-tool
+          key: ${{ runner.os }}-cve-bin-tool-${{ steps.get-date.outputs.date }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install -r doc/requirements.txt
+      - name: Install cabextract
+        run: sudo apt-get install cabextract
+      - name: try single cli run of tool
```

Comment:
If you drop writing .csv file to cache this step can be dropped as well.

Reply:
Hmm, yeah. I agree that I could remove quite a few things if I drop writing in cache.
If I'm doing that where will it be better to place the .csv file?
In the project directory?

Also a silly doubt, Aren't we using cache so that the files will be cleaned up? If so then isn't it better to keep it this way?

Reply:
Anywhere in the project directory is fine, it will be discarded after the job finishes anyway.

Hmm, I don't quite understand, what do you mean to clean files up? We use cache to speed up things.

Reply:
Thanks!
> Hmm, I don't quite understand, what do you mean to clean files up? We use cache to speed up things.

Sorry. That was a misunderstanding I had